ci/test-clang.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -eu
+
+trap 'rm -fr "$toolchain"' EXIT
+
+# Set up workspace
+rootdir=${GITHUB_WORKSPACE:-$(dirname "$(dirname "$(realpath "$0")")")}
+toolchain=$rootdir/toolchain
+
+# Pull toolchain out of container
+echo "[+] Downloading toolchain from container"
+docker create --name llvm-project ghcr.io/clangbuiltlinux/llvm-project:stage2
+mkdir -p "$toolchain"/{bin,include,lib}
+docker cp llvm-project:/usr/local/bin/. "$toolchain"/bin
+docker cp llvm-project:/usr/local/include/. "$toolchain"/include
+docker cp llvm-project:/usr/local/lib/. "$toolchain"/lib
+echo "[+] Cleaning up container"
+docker rm llvm-project
```

Comment:
in a follow up commit, we might want to do this as one of the final build steps. We probably don't need to republish the llvm sources (or tarball). Having a working shell is handy though!

Reply:
I am confused, isn't this just removing the container I created?

Reply:
oh, yeah, I see.  Sorry, I was thinking about the llvm-project sources within the container. nvm